src/d3a/d3a_core/rq_job_handler.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import ast
+import json
+import pickle
+from datetime import datetime, date
+from pendulum import duration, instance
+from zlib import decompress
+
+from d3a.models.config import SimulationConfig
+from d3a.d3a_core.simulation import run_simulation
+from d3a_interface.constants_limits import GlobalConfig, ConstSettings
+from d3a_interface.settings_validators import validate_global_settings
+import d3a.constants
+from d3a.d3a_core.util import available_simulation_scenarios, update_advanced_settings
+
+
+log = logging.getLogger()
+
+
+def decompress_and_decode_queued_strings(queued_string):
+    return pickle.loads(decompress(queued_string))
+
+
+def launch_simulation_from_rq_job(scenario, settings, events, aggregator_device_mapping,
+                                  saved_state, job_id):
+    logging.getLogger().setLevel(logging.ERROR)
+    scenario = decompress_and_decode_queued_strings(scenario)
+    if "collaboration_uuid" in scenario:
+        d3a.constants.COLLABORATION_ID = scenario.pop("collaboration_uuid")
+        d3a.constants.EXTERNAL_CONNECTION_WEB = True
+        d3a.constants.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
+        d3a.constants.RUN_IN_REALTIME = d3a.constants.IS_CANARY_NETWORK
+    saved_state = decompress_and_decode_queued_strings(saved_state)
+    log.error(f"Scenario: {scenario}")
+    log.error(f"Settings: {settings}")
+    log.error(f"Events: {events}")
+    log.error(f"Aggregator device mapping: {aggregator_device_mapping}")
+    log.error(f"Previous simulation state: {saved_state}")
```

Comment:
Cant these all be squashed into single error log?

Reply:
Sure, merging these, thanks.

Reply:
I am voting for leaving these as is. Each of them are huge for complex sims. So either you add some `\n` in there leave these in order to separate the information. 
I hope you agree.

Reply:
Ah just saw your commit. All good.